path_config.go

```diff
@@ -37,13 +38,45 @@ clears the current setting. Accepts a Go duration format string.`,
 			logical.UpdateOperation: &framework.PathOperation{
 				Callback: b.upgradeCheck(b.pathConfigWrite()),
 				Summary:  "Configure backend level settings that are applied to every key in the key-value store.",
+				Responses: map[int][]framework.Response{
+					http.StatusNoContent: {{
+						Description: http.StatusText(http.StatusNoContent),
+					}},
+				},
 			},
 			logical.CreateOperation: &framework.PathOperation{
 				Callback: b.upgradeCheck(b.pathConfigWrite()),
+				Responses: map[int][]framework.Response{
+					http.StatusNoContent: {{
+						Description: http.StatusText(http.StatusNoContent),
+					}},
+				},
 			},
 			logical.ReadOperation: &framework.PathOperation{
 				Callback: b.upgradeCheck(b.pathConfigRead()),
 				Summary:  "Read the backend level settings.",
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: http.StatusText(http.StatusOK),
+						Fields: map[string]*framework.FieldSchema{
+							"max_versions": {
+								Type:        framework.TypeInt,
+								Description: "The number of versions to keep for each key. Defaults to 10",
+								Required:    true,
+							},
+							"cas_required": {
+								Type:        framework.TypeBool,
+								Description: "If true, the backend will require the cas parameter to be set for each write",
+								Required:    true,
+							},
+							"delete_version_after": {
+								Type:        framework.TypeSignedDurationSecond,
+								Description: "If set, the length of time before a version is deleted.",
```

Comment:
the description seems to indicate this field is optional?

Reply:
I copy pasted it from the request description I think 😄 

```suggestion
								Description: "The length of time before a version is deleted.",
```

Reply:
The field is present in every successful response:
https://github.com/hashicorp/vault-plugin-secrets-kv/blob/d0f28226c69b9b9fbc1d01383a24e0298f0c0640/path_config.go#L95-L112